P10 Added judging critera page

andrewdempsey2018 opened the pull request:
Added basic "Judging Criteria" page with some placeholder text. Setup urls, view and navbar link to access the page.

stefdworschak commented on .env_sample:
```diff
@@ -1,3 +0,0 @@
-DEVELOPMENT=1
```
Why did you delete the file? Could you please add this back in as this is for all students? If you don't need it locally, you can delete it, but please don't commit the change.

TravelTimN replied:
File added back in - resolving conversation

stefdworschak commented on home/templates/home/criteria.html:
```diff
@@ -0,0 +1,49 @@
+
+{% block content %}
+
+<H1>Judging Criteria for Hackaton</H1>
```
Please change this to lowercase `<h1>`

stefdworschak commented on home/templates/home/criteria.html:
```diff
@@ -0,0 +1,49 @@
+{% block content %}
+
+<H1>Judging Criteria for Hackaton</H1>
+<br>
```
In general I would avoid line breaks. Could you remove them and use bootstrap grid instead?

stefdworschak commented on home/templates/home/criteria.html:
```diff
@@ -0,0 +1,49 @@
+    <li>Overall Experience</li>
+</ul>
+<br>
+<h2><b>Content</b></h2>
```
Could you please replace the `<b>` tags with `<strong>` . Alternatively, you could create a custom css class called `bold` (or similar) and add the class to the `<h2>`.

stefdworschak commented on home/templates/home/criteria.html:
```diff
@@ -0,0 +1,49 @@
+    medium. Good content takes a stand. It has a voice and a point of view. It may be informative, useful, or funny, but
+    it always leaves you wanting more.</p>
+<br>
+<h2><b>Structure and navigation</b></h2>
```
Same here (and for all of the instances of `<b>`)

stefdworschak commented on home/views.py:
```diff
@@ -5,3 +5,8 @@ def index(request):
     """ A view to return the index page """
 
     return render(request, "home/index.html")
+
```
We are following PEP8 standard where you would want 2 line breaks between functions or classes. Could you please add another line break here? If you are interested, you can read more here: https://www.python.org/dev/peps/pep-0008/

stefdworschak reviewed (changes requested):
@andrewdempsey2018 this is a really good first pull request you can be proud of! Just a few minor changes that I would recommend. Once you made the changes, just tag me or Tim in the Pull Request again and we can approve it. For future use I would also recommend to flesh out the Pull Request message a little bit (e.g. add areas such as Description, Tests, Errors, Screenshots, etc.); we will also add a template shortly as well.

TravelTimN commented on home/templates/home/criteria.html:
```diff
@@ -0,0 +1,69 @@
+    </div>
+
+</div>
+{% endblock %}
```
@andrewdempsey2018 
needs a new line at the end of the file

andrewdempsey2018 commented:
@stefdworschak @TravelTimN 

User Story ID: #P10

Description of PR:
Added a basic Judging Criteria page with dummy information in lieu of a final writeup of criteria. Page contains four locations of information with headings on a 2x2 grid & is responsive for small screens. Home/urls.py reflects the new page as well as home/views.py

Tests:

- Page has been tested on desktop browsers Firefox and Chrome. It has also been tested on an emulated smartphone.

Screenshots:

Desktop Firefox test:

![image](https://user-images.githubusercontent.com/38543181/96355056-76ac1680-10d5-11eb-867d-35db8a3260ea.png)

Desktop Chrome test:

![image](https://user-images.githubusercontent.com/38543181/96355067-89265000-10d5-11eb-9cc9-a9c28ff5daa8.png)

Emulated smartphone test:

![image](https://user-images.githubusercontent.com/38543181/96355074-96dbd580-10d5-11eb-84cc-c1c57e8e284f.png)

Know bugs/errors:

- Page needs actual content so actual judging critera needs to be written up.

Post review/additional changes:

- reinstated .env_sample after deleting it accidentally
- rewrote some tags with lower case to keep HTML code style consistent
- updated home>views.py to be in line with PEP8 standard
- added new line at the end of criteria.html

stefdworschak reviewed (approved):
LGTM

Just FYI - not sure if you are aware, but you can edit the PR message body. Also, usually if somebody leaves comments, you would use the Reply option for each comment and Resolve the conversation once the comment is addressed. Not a biggie though.

TravelTimN reviewed (approved):
LGTM - might get @stefdworschak to just check from his end.
Also - there might be some issues, since @JimLynx was working on the footer, and the view/url for this was different.
I do like @andrewdempsey2018 namespace better though, short and sweet.

JimLynx commented:
> LGTM
> 
> Just FYI - not sure if you are aware, but you can edit the PR message body. Also, usually if somebody leaves comments, you would use the Reply option for each comment and Resolve the conversation once the comment is addressed. Not a biggie though.

Noted, thank you. Will ensure I do so in future.

TravelTimN commented:
Actually, @andrewdempsey2018 - can you have a look at #74  and adjust the urls/views accordingly to match that of @JimLynx  please?
`judging_criteria` instead of just `criteria`, as this could eventually be expanded and have a different meaning for something else later. Thanks!

andrewdempsey2018 commented:
> judging_criteria

andrewdempsey2018 commented:
> 
> 
> LGTM
> 
> Just FYI - not sure if you are aware, but you can edit the PR message body. Also, usually if somebody leaves comments, you would use the Reply option for each comment and Resolve the conversation once the comment is addressed. Not a biggie though.

Noted, thanks for the tip.

stefdworschak reviewed (approved):
LGTM
